wgs.yaml

```diff
@@ -491,32 +387,11 @@ workinggroups:
     - name: wg-training-reviews
       description: API Changes and Reviews (API Machinery APIs, NOT all APIs)
   subprojects:
-  - name: caffe2-operator
-    owners:
-    - https://raw.githubusercontent.com/kubeflow/caffe2-operator/master/OWNERS
-  - name: chainer-operator
-    owners:
-    - https://raw.githubusercontent.com/kubeflow/chainer-operator/master/OWNERS
-  - name: common
-    owners:
-    - https://raw.githubusercontent.com/kubeflow/common/master/OWNERS
-  - name: fate-operator
-    owners:
-    - https://raw.githubusercontent.com/kubeflow/fate-operator/master/OWNERS
   - name: mpi-operator
     owners:
-    - https://raw.githubusercontent.com/kubeflow/mpi-operator/master/OWNERS
-  - name: mxnet-operator
-    owners:
-    - https://raw.githubusercontent.com/kubeflow/mxnet-operator/master/OWNERS
-  - name: pytorch-operator
-    owners:
-    - https://raw.githubusercontent.com/kubeflow/pytorch-operator/master/OWNERS
-  - name: tf-operator
-    owners:
-    - https://raw.githubusercontent.com/kubeflow/tf-operator/master/OWNERS
-  - name: xgboost-operator
+    - https://github.com/kubeflow/mpi-operator/blob/master/OWNERS
+  - name: training-operator
```

Comment:
I would post-pone this change until we archive all other Distributed Training repos (e.g. Fate Operator: https://github.com/kubeflow/fate-operator)